nightly build examples all have: OF_ROOT = /home/ofadmin

The release version of OF isn't compatible with the release version of make (something about missing seperator, I saw an issue, it is fixed in master, which is why I am using nightly...

Which has this issue:

```
carl@twist:~/of/of_v20201229_linux64gcc6_release/examples/gl/vboExample$ make
Makefile:13: /home/ofadmin/openFrameworks/scripts/dev/openFrameworks_pkg_creation/libs/openFrameworksCompiled/project/makefileCommon/compile.project.mk: No such file or directory
make: *** No rule to make target '/home/ofadmin/openFrameworks/scripts/dev/openFrameworks_pkg_creation/libs/openFrameworksCompiled/project/makefileCommon/compile.project.mk'.  Stop.
```
seems all of the examples have 
`config.make:11:OF_ROOT = /home/ofadmin`


```
wget https://openframeworks.cc/ci_server/versions/nightly/of_v20201229_linux64gcc6_nightly.tar.gz
...
carl@twist:~/of/of_v20201229_linux64gcc6_release/examples$ ack ofadmin|head
3d/pointCloudExample/config.make:11:OF_ROOT = /home/ofadmin/openFrameworks/scripts/dev/openFrameworks_pkg_creation
3d/cameraRibbonExample/config.make:11:OF_ROOT = /home/ofadmin/openFrameworks/scripts/dev/openFrameworks_pkg_creation
3d/cameraLensOffsetExample/config.make:11:OF_ROOT = /home/ofadmin/openFrameworks/scripts/dev/openFrameworks_pkg_creation
3d/assimpExample/config.make:11:OF_ROOT = /home/ofadmin/openFrameworks/scripts/dev/openFrameworks_pkg_creation
3d/normalsExample/config.make:11:OF_ROOT = /home/ofadmin/openFrameworks/scripts/dev/openFrameworks_pkg_creation
3d/ofBoxExample/config.make:11:OF_ROOT = /home/ofadmin/openFrameworks/scripts/dev/openFrameworks_pkg_creation
3d/3DPrimitivesExample/config.make:11:OF_ROOT = /home/ofadmin/openFrameworks/scripts/dev/openFrameworks_pkg_creation
3d/pointPickerExample/config.make:11:OF_ROOT = /home/ofadmin/openFrameworks/scripts/dev/openFrameworks_pkg_creation
3d/orientationExample/config.make:11:OF_ROOT = /home/ofadmin/openFrameworks/scripts/dev/openFrameworks_pkg_creation
3d/ofNodeExample/config.make:11:OF_ROOT = /home/ofadmin/openFrameworks/scripts/dev/openFrameworks_pkg_creation
```

--- a/scripts/dev/create_package.sh
+++ b/scripts/dev/create_package.sh
@@ -195,6 +195,16 @@ function createProjectFiles {
             pg_platform=$pkg_platform
         fi
         ${main_ofroot}/apps/projectGenerator/commandLine/bin/projectGenerator_debug --recursive -p${pg_platform} -o$pkg_ofroot $pkg_ofroot/examples > /dev/null
+        
+        #fix config.make because the project generator is putting in the full path to the OF_ROOT as it is designed to do.
+        #in this case we actually don't want to set it as the default of ../../../ is fine.
+        for example_group in $pkg_ofroot/examples/*; do
+            for example in $example_group/*; do
+                if [ -d $example ]; then
+                    sed -i 's/^OF_ROOT =.*/# OF_ROOT = ..\/..\/..\//' $example/config.make
+                fi
+            done
+        done
     elif [ "$pkg_platform" == "linuxarmv6l" ] || [ "$pkg_platform" == "linuxarmv7l" ]; then
         for example_group in $pkg_ofroot/examples/*; do
             for example in $example_group/*; do

